openid-sharedsignals-framework-1_0.md

```diff
@@ -0,0 +1,2169 @@
+    "urn:example:secevent:events:type_2",
+    "urn:example:secevent:events:type_3"
+  ],
+  "events_requested": [
+    "urn:example:secevent:events:type_2",
+    "urn:example:secevent:events:type_3",
+    "urn:example:secevent:events:type_4"
+  ],
+  "events_delivered": [
+    "urn:example:secevent:events:type_2",
+    "urn:example:secevent:events:type_3"
+  ]
+}
+~~~
+{: title="Example: Replace Stream Configuration Response" #figreplaceconfigresp}
+
+Pending conditions or errors are signaled with HTTP status codes as follows:
+
+| Code | Description |
+|------|-------------|
+| 202  | if the replace request has been accepted, but not processed. Receiver MAY try the same request later in order to get processing result. |
+| 400  | if the request body cannot be parsed, a Transmitter-Supplied property is incorrect, or if the request is otherwise invalid |
+| 401  | if authorization failed or it is missing |
+| 403  | if the Event Receiver is not allowed to replace the stream configuration |
+| 404  | if there is no Event Stream with the given "stream_id" for this Event Receiver |
+{: title="Replace Stream Configuration Errors" #tabreplaceconfig}
+
+#### Deleting a Stream {#deleting-a-stream}
+An Event Receiver deletes a stream by making an HTTP DELETE request to the
+Configuration Endpoint. On receiving a request the Event Transmitter responds
+with an empty `204 OK` response if the configuration was successfully removed.
+
+The DELETE request MUST include the "stream_id" as a parameter in order to
+identify the correct Event Stream.
+
+The following is a non-normative example request to delete an Event Stream:
+
+~~~ http
+DELETE /sse/stream?stream_id=f67e39a0a4d34d56b3aa1bc4cff0069f HTTP/1.1
+Host: transmitter.example.com
+Authorization: Bearer eyJ0b2tlbiI6ImV4YW1wbGUifQo=
+~~~
+{: title="Example: Delete Stream Request" #figdeletestreamreq}
+
+Errors are signaled with HTTP status codes as follows:
+
+| Code | Description |
+|------|-------------|
+| 401  | if authorization failed or it is missing |
+| 403  | if the Event Receiver is not allowed to delete the stream |
+| 404  | if there is no Event Stream with the given "stream_id" for this Event Receiver |
+{: title="Delete Stream Errors" #tabdeletestream"}
+
+### Stream Status {#status}
+Within an Event Stream, events related to different Subject Principals MAY be
+managed independently. A Receiver MAY request Subject Principals to be added to
+or removed from a stream by Updating the Stream Status
+({{updating-a-streams-status}}) and specifying the Subject in the request.
+
+A Transmitter MAY decide to enable, pause or disable updates about a Subject
+independently of an update request from a Receiver. If a Transmitter decides to
+start or stop events for a Subject then the Transmitter MUST do the following
+according to the status of the stream.
+
+If the stream is:
+
+Enabled
+: the Transmitter MUST send a stream updated ({{stream-updated-event}}) event
```

Comment:
Should we use the newline and `>` format that we use for all of the other definitional lists in this document? (see just below on lines 1381 - 1401)

Reply:
done.